src/index.js

```diff
@@ -68,6 +68,19 @@ const merge = (policies) => {
   return {rules: result};
 };
 
+const extract = (policy, privilege, attribute) => {
+  validate(policy);
+  return Object.entries(policy.rules).map(([operation, rules]) => {
+    if (Array.isArray(rules) && privilege === operation) {
+      for (const rule of rules) {
+        if (rule[attribute]) {
+          return rule[attribute];
```

Comment:
There could be multiple rules conditions that reference the same attribute, so this needs to collect all the rule conditions that it finds, rather then just returning the first.